src/DotLiquid.Tests/Util/StrFTimeTests.cs

```diff
@@ -39,12 +38,30 @@ public class StrFTimeTests
         [TestCase("%y", ExpectedResult = "12")]
         [TestCase("%Y", ExpectedResult = "2012")]
         [TestCase("%", ExpectedResult = "%")]
-        public string TestFormat(string format)
-        {
-            using (CultureHelper.SetCulture("en-GB"))
-            {
+        public string TestFormat(string format) {
+            using (CultureHelper.SetCulture("en-GB")) {
+                Assert.That(CultureInfo.CurrentCulture, Is.EqualTo(new CultureInfo("en-GB")));
+                var date = new DateTime(2012, 1, 8, 14, 32, 14);
+                var localResult = date.ToStrFTime(format);
+                var utcResult = new DateTimeOffset(date, TimeSpan.FromHours(0)).ToStrFTime(format);
+                Assert.AreEqual(localResult, utcResult);
+                var estResult = new DateTimeOffset(date, TimeSpan.FromHours(-5)).ToStrFTime(format);
+                Assert.AreEqual(utcResult, estResult);
+                return localResult;
+            }
+        }
+
+        [Test]
+        public void TestEpoch() {
+            using (CultureHelper.SetCulture("en-GB")) {
                 Assert.That(CultureInfo.CurrentCulture, Is.EqualTo(new CultureInfo("en-GB")));
-                return new DateTime(2012, 1, 8, 14, 32, 14).ToStrFTime(format);
+                var date = new DateTime(2012, 1, 8, 14, 32, 14);
+                var localResult = date.ToStrFTime("%s");
+                Assert.AreEqual("1326033134", localResult);
+                var utcResult = new DateTimeOffset(2012, 1, 8, 14, 32, 14, TimeSpan.FromHours(0)).ToStrFTime("%s");

```

Comment:
nit: I should read my code, could just pass date instead of 6 params.

Reply:
updating now

Reply:
Thanks for updating, I was not planning on waiting for those changes just waiting for the build. We should have a nuget build shortly. Thanks again for your contribution and patience!

Reply:
Of course! Thank you for merging it in.

We have our own fork of the project for some internal macro logic but this seemed like something that should go upstream ;)